assets/js/blocks/price-filter/block.tsx

```diff
@@ -116,12 +135,18 @@ const PriceFilterBlock = ( { attributes, isEditor = false } ) => {
 	);
 
 	const { minConstraint, maxConstraint } = usePriceConstraints( {
-		minPrice: results.price_range
-			? results.price_range.min_price
-			: undefined,
-		maxPrice: results.price_range
-			? results.price_range.max_price
-			: undefined,
+		minPrice:
+			objectHasProp( results, 'price_range' ) &&
+			objectHasProp( results.price_range, 'min_price' ) &&
+			isString( results.price_range.min_price )
+				? results.price_range.min_price
+				: undefined,
+		maxPrice:
+			objectHasProp( results, 'price_range' ) &&
+			objectHasProp( results.price_range, 'max_price' ) &&
+			isString( results.price_range.max_price )
+				? results.price_range.max_price
+				: undefined,
```

Comment:
I don't like these long type guards but I haven't figured out how to write recursive type predicates 😬.